### DISCUSSION TITLE
Is it possible to get another audio format?

### DISCUSSION THREAD

**Discussion by bybpow** [subdavis/kobo-book-downloader#49] [Q&A] [answered]
Right now the tool just downloads mp3s. Is there a way to download any other format from source like acc or ogg?

**Comment by subdavis** [comment, marked as answer]
audio transcoding is well out of scope for this tool

I think you're looking for `ffmpeg`

`ffmpeg -i filename.mp3 newfilename.ogg`

**Reply by bybpow** [reply]
What I meant is if the original source its always an mp3. The tools is great and I don't think it needs trascoding. I just wondered if there are more formats in the original source stream instead of converting after downloading. Thanks for your answer.

**Reply by subdavis** [reply]
Oh, I see what you mean: can you force the server to give you back a different file.    I have no idea, but I sorta doubt it.  This is a reverse engineering effort, so you're welcome to take a look at the information coming out of kobo and see if you can tweak parameters to get other audio formats.  I probably won't spend time looking into this, but PRs are welcome.
